src/main/resources/resources.conf

```diff
@@ -0,0 +1,14 @@
+resources {
```

Comment:
I don't think this should be in a separate file, put it in reference.conf for now